### ISSUE TITLE
Energized sources and overblocking.

### ISSUE THREAD

**Issue by ace2k** [EnergizedProtection/block#360]
After encountering several legit domains being blocked since the creation of my [last report](https://github.com/EnergizedProtection/block/issues/355), I've started conducting some tests with the sources used for the "Ultimate" list and discovered a severe case of overblocking with more than 1200  domains from some plumbing business, over localized video streaming platforms to even [OVH's own speedtest](http://proof.ovh.net). Most of these issues stem from the same less known, less reputable lists, that all seem to suffer the same issue of what looks to be arbitrary blocking of legitimate websites.

I will refrain from reporting every single falsely blocked domain here as I believe it's completely out of scope _(especially since you have to check them yourself and confirm)_ and most likely the list wouldn't be even near complete as I've only conducted rather superficial testing with the top 10k domains based on alexa.com ranking and similar meta information aggregators.

Consolidating malicious domain lists while filtering out duplicates is definitely a good idea as not every software has this ability and often ends up querying all entries from all subscribed lists - including countless duplicates, which ultimately results in poor performance, especially on mobile devices. Nontheless I believe that stuffing every single filterlist ever found on the web is going to create the main issue described above.

The whitelist effort is an attempt to overcome this issue, but it's always going to stay a chicken chase where the sources keep blocking legitimate domains only for people to report them here in an attempt to get whitelisted. An unrealistic effort compared against the potential gain.

I will be returning to [my previous setup](https://gist.github.com/ace2k/41d41c8f150b47286a48947618c56413) I used for the Pi-Hole consisting mostly of widely-known, curated lists for the time being, which amount to about 350k entries. At the end of the day, you want to get rid of trackers, ads and stop software from phoning home in a sketchy fashion (_Hey Microsoft! 🙋‍♂️_) rather than censoring your own access to the internet. My suggestion for this project is to consider the same principle in the future and priorize quality over quantity.

Thanks for reading!

**Comment by owah** [comment]
I completely agree, I just started using this list a few days ago, and I submitted 2 issues already to unblock 2 CDNs. I’ve noticed some more false positives, but I’ve not reported them yet, as to not spam the issue tracker.

**Comment by AdroitAdorKhan** [comment]
@ace2k Thanks for the suggestion! As you may know, we have smaller lists like Blu/BluGo/Spark if Ultimate/Basic is blocking too much for you.

But for the issues you have submitted, they are from the sources of other maintainer. I have already knocked the maintainer about the issues, so they might fix them asap! Moreover, I will stop using the source(s) which block too many legit domains. I have already sorted these false/positive issues in spark, blugo and blu; by removing few sources. Packs larger than blu might still have these issues because they are basically for higher end devices (ex. PCs), not mobile devices. As we browse more on PCs than mobile devices. Also, I have made submitting issues more easier via [Website](https://app.energized.pro) or [Telegram Direct Message](https://t.me/AdroitAdorKhan) besides Github issues. I have been working on this project almost everyday to whitelist/blacklist user submitted domains or the domains I come across while browsing. I just need you guys to submit them to us, to make this a better project for all!

**Comment by durablenapkin** [comment]
@AdroitAdorKhan thanks for that, I noticed that blogto.com isn't blocked anymore which is a huge improvement.

> Moreover, I will stop using the source(s) which block too many legit domains

This is what https://github.com/StevenBlack/hosts/ does, see: https://github.com/StevenBlack/hosts/commit/0d4c7b800a75d6cc4b777f8790da256dbcd0a009

**Comment by It0w** [comment]
A good example for overblocking:
`your-server.de` = one of the biggest server hosting in germany "great idea to block the entire domain -.-"
`herokudns.com` = is used for many Indi games like Factorio for example

Please dont block entire domains if not nacescary

**Comment by DandelionSprout** [comment]
I'm probably far too late to chip in on this, but I personally feel that any active use of Ultimate or Unified by end-users is equivalent to a deathwish mission. If we include all the sources of 1Hosts separately, Ultimate has north of 300 sources, and Unified has more than 310. Many of those 310 are smalltime sideproject lists that occasionally go far beyond normal customs in their blocking of privacy intruders. This makes it nigh-on impossible to figure out which source list that has added a specific odd-looking entry.

At most I could've supported the use of Blu Go + Regional.
